contrib/systemd/asterisk.service

```diff
@@ -16,19 +17,51 @@ Group=asterisk
 ExecStart=/usr/sbin/asterisk -mqf -C /etc/asterisk/asterisk.conf
```

Comment:
Oh you know what?  I didn't expand the whole diff and forgot about
```
User=asterisk
Group=asterisk
```
Two issues...  

There's nothing in an asterisk install that creates that user and group.  It's totally up to the packager.

and...

When asterisk.conf has runuser specified, asterisk knows it's not going to be run as a 
privileged user so, thinking it was actually started as root as it would have been in the old days, it tries to get the `cap_net_admin` capability before switching the user.  If the service specifies User then that will fail and you'll see a message like `Unable to install capabilities.` almost immediately after startup but startup will continue.  If the service specifies User and runuser isn't in asterisk.conf then we don't even try.  IIRC, at one time we needed that capability to set QOS on packets when not running as a privileged user.  I _think_ that may still be the case.

So, I'm wondering if we should leave User and Group commented out but add a comment explaining that setting runuser and rungroup in asterisk.conf is preferred for the reason above.

Having said all of that, the proper systemd way to do this would probably be adding AmbientCapabilities=CAP_NET_ADMIN to the file but I've never tried it so I don't know if it works or not.